src/i18n/strings/en_EN.json

```diff
@@ -1830,6 +1831,15 @@
         "other": "%(count)s members",
         "one": "%(count)s member"
     },
+    "%(names)s and %(name)s": "%(names)s and %(name)s",
+    "%(names)s and %(count)s others": {
+        "other": "%(names)s and %(count)s others",
+        "one": "%(names)s and %(count)s other"
+    },
+    "%(count)s people asking to join": {
+        "other": "%(count)s people asking to join",
+        "one": "Asking to join"
```

Comment:
This will never happen, so it's slightly odd, but I guess it's good for languages that have several different plural forms to make it clear here that this is a plural.

Reply:
Oh no, ignore me - it does get used :-)